.yamato/pytest-gpu.yml

```
@@ -0,0 +1,21 @@
+pytest_gpu:
+  name: Pytest GPU
+  agent:
+    type: Unity::VM::GPU
+    image: package-ci/ubuntu:stable
+    flavor: b1.large
+  commands:
+    - |
+      sudo apt-get update && sudo apt-get install -y python3-venv
+      python3 -m venv venv && source venv/bin/activate
+      python3 -m pip install pyyaml --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+      python3 -u -m ml-agents.tests.yamato.setup_venv
+      python3 -m pip install --progress-bar=off -r test_requirements.txt --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+      python3 -m pip install torch==1.7.1+cu101 torchvision==0.8.2+cu101 torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+      python3 -m pytest -m "not check_environment_trains" --cov=ml-agents --cov=ml-agents-envs --cov=gym-unity --cov-report html --junitxml=junit/test-results-${{ matrix.python-version }}.xml -p no:warnings
```

```suggestion
      python3 -m pytest -m "not check_environment_trains" --junitxml=junit/test-results.xml -p no:warnings
```
Don't need the coverage options (since we're only running a subset and don't want to fail for low coverage) and `matrix.python-version` is part of the github action definition.

---

Might want to make `junit` an artifact directory so the file is saved (not sure it'll be useful though).

I think something like this is all you need https://github.com/brccabral/ml-agents/blob/efa55731c689e6abad062a64509c51be6f936351/.yamato/training-int-tests.yml#L33-L36